python/mxnet/rnn/rnn_cell.py

```diff
@@ -404,7 +409,7 @@ def __call__(self, inputs, states):
                                           name="%sslice"%name)
         in_gate = symbol.Activation(slice_gates[0], act_type="sigmoid",
                                     name='%si'%name)
-        forget_gate = symbol.Activation(slice_gates[1], act_type="sigmoid",
+        forget_gate = symbol.Activation(slice_gates[1] + self._forget_bias, act_type="sigmoid",
```

Comment:
Ok, I see but does using "bias = mx.Variable('_bias', init=LSTMbias)" solve it? 

So far, I only see one way to do it by relying on a forget bias name convention in FusedRNN._init_bias but it is ugly. Do you have a better idea in mind?